docs/index.bs

```diff
@@ -1576,11 +1576,16 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       The <dfn method for="InstallEvent"><code>addRoutes(|rules|)</code></dfn> method steps are:
 
         1. Let |routerRules| be a list of {{RouterRule}} dictionaries.
+        1. Let |hasFetchEventSource| be a boolean, initially false.
+        1. Let |serviceWorker| be the [=current global object=]'s associated [=ServiceWorkerGlobalScope/service worker=].
+        1. If |serviceWorker|'s [=list of router rules=] [=list/contains=] a {{RouterRule}} whose {{RouterRule/source}} is "{{RouterSource/fetch-event}}", set |hasFetchEventSource| to true.
         1. If |rules| is a {{RouterRule}} dictionary, set |rules| to &#x00AB; |rules| &#x00BB;.
         1.  For each |rule| of |rules|:
-            1. If running [=Verify Router Rule=] algorithm with |rule| and [=/service worker=] returns false, <a>throw</a> a <code>TypeError</code>.
+            1. If running [=Verify Router Rule=] algorithm with |rule| and [=/service worker=] returns false, [=throw=] a {{TypeError}}.
```

Comment:
I am feeling guilty on not doing like so, but this [=/service worker=] might be |serviceWorker|.

Reply:
done